app/src/main/java/com/github/weg_li_android/MainActivity.kt

```diff
@@ -25,12 +59,177 @@ class MainActivity : AppCompatActivity() {
             ViewModelFactory(ApiHelper(ApiServiceImpl()))
         ).get(MainViewModel::class.java)
 
+        setupPhotoCard(this)
+
         setupCarTypeSpinner()
         setupViolationSpinner()
         durationText.setOnClickListener {
         }
 
         sendButton.setOnClickListener { mainViewModel.sendReport() }
+
+        val licenseObserver = Observer<Report> { newName ->
+            if(newName.license != "") {
```

Comment:
Please reformat the file (use Android Studio's builtin way to do that). Also, you can use `isEmpty` or `isNotEmpty` here.